### DISCUSSION TITLE
Can you use a combination of Sling, git and CSV data to do DevOps CI/CD database data versioning ?

### DISCUSSION THREAD

**Discussion by alberttwong** [slingdata-io/sling-cli#167] [General]
If I had a CI/CD pipeline and I wanted to re-stage the database with the same data set for Unit Testing, do you think Sling with Git and CSV data can solve this use case?

**Comment by flarco** [comment]
For sure, a replication can be defined with all the relative files in Git.
The only concern would be the size of the data for git (not for sling), would probably have to be kept fairly small.

**Reply by flarco** [reply]
There is also the Github Action Component: https://github.com/marketplace/actions/sling-run-action
This makes it easy to deploy in GA for CI/CD
